### ISSUE TITLE
everyvoice/run_tests.py all   (failed)    many   pydantic / G2P fails.  ( No reports at end wihout  --verbose)

### ISSUE THREAD

**Issue by marctessier** [EveryVoiceTTS/EveryVoice#763]
### Bug description

While testing the latest version of EV  ( 2025-11-13)  I was noticing  a strange behaviour.

No reports were being generated at the end of the test .  I think it is crashing. ( seeing rc=1 )

On Trixie,  the tested ended with this below: Noticed the `%` at end... That's not normal

`Create a EveryVoiceConfig which pydantic will validate for us. ... %`

Also on Trixie,  I noticed when I ran the test on the head node. It "core dumped ". with below the last line.   
(This could be OK , possibly where on TRIXIE a new policy is in place   where processes are being killed if too much resources are being used..)

`test_evaluate (everyvoice.tests.test_cli.CLITest.test_evaluate) ... CPU time limit exceeded (core dumped)`

When I run --verbose   , we get better details of the issue.. See attached  error.txt  for the full error.

```
Ran 280 tests in 61.030s

FAILED (failures=1, errors=13, skipped=1)
```

[error.txt](https://github.com/user-attachments/files/23611527/error.txt)

### How to reproduce the bug

Install the latest version of EV and run all the tests.

### Error messages and logs

```
# Error messages and logs here please
```

### Environment

<details>
<summary>Current environment</summary>
<pre>
(EveryVoice_2025-11-18) $ everyvoice --diagnostic
EveryVoice Diagnostic information
EveryVoice version: 0.4.0
Python version: 3.12.12 | packaged by conda-forge | (main, Oct 22 2025, 23:34:53) [Clang 19.1.7 ]
System: Darwin 24.6.0 Darwin Kernel Version 24.6.0: Mon Jul 14 11:30:51 PDT 2025; root:xnu-11417.140.69~1/RELEASE_ARM64_T8112 arm64
Environment type: conda

Modules installed using conda:
# packages in environment at /Users/tessierm/miniconda3/envs/EveryVoice_2025-11-18:
#
# Name                    Version                   Build  Channel
aom                       3.9.1                h7bae524_0    conda-forge
bzip2                     1.0.8                hd037594_8    conda-forge
ca-certificates           2025.11.12           hbd8a1cb_0    conda-forge
cairo                     1.18.4               h6a3b0d2_0    conda-forge
dav1d                     1.2.1                hb547adb_0    conda-forge
dbus                      1.16.2               hda038a8_0    conda-forge
ffmpeg                    8.0.0           gpl_h670d5b4_106    conda-forge
font-ttf-dejavu-sans-mono 2.37                 hab24e00_0    conda-forge
font-ttf-inconsolata      3.000                h77eed37_0    conda-forge
font-ttf-source-code-pro  2.038                h77eed37_0    conda-forge
font-ttf-ubuntu           0.83                 h77eed37_3    conda-forge
fontconfig                2.15.0               h1383a14_1    conda-forge
fonts-conda-ecosystem     1                             0    conda-forge
fonts-conda-forge         1                    hc364b38_1    conda-forge
freetype                  2.14.1               hce30654_0    conda-forge
fribidi                   1.0.16               hc919400_0    conda-forge
gdk-pixbuf                2.44.4               h7542897_0    conda-forge
gettext                   0.25.1               h3dcc1bd_0    conda-forge
gettext-tools             0.25.1               h493aca8_0    conda-forge
glslang                   16.0.0               h60b4770_0    conda-forge
gmp                       6.3.0                h7bae524_2    conda-forge
graphite2                 1.3.14               hec049ff_2    conda-forge
harfbuzz                  12.2.0               haf38c7b_0    conda-forge
icu                       75.1                 hfee45f7_0    conda-forge
lame                      3.100             h1a8c8d9_1003    conda-forge
lerc                      4.0.0                hd64df32_1    conda-forge
libabseil                 20250512.1      cxx17_hd41c47c_0    conda-forge
libasprintf               0.25.1               h493aca8_0    conda-forge
libasprintf-devel         0.25.1               h493aca8_0    conda-forge
libass                    0.17.4               hcbd7ca7_0    conda-forge
libcxx                    21.1.6               hf598326_0    conda-forge
libdeflate                1.25                 hc11a715_0    conda-forge
libexpat                  2.7.1                hec049ff_0    conda-forge
libffi                    3.5.2                he5f378a_0    conda-forge
libflac                   1.4.3                hb765f3a_0    conda-forge
libfreetype               2.14.1               hce30654_0    conda-forge
libfreetype6              2.14.1               h6da58f4_0    conda-forge
libgettextpo              0.25.1               h493aca8_0    conda-forge
libgettextpo-devel        0.25.1               h493aca8_0    conda-forge
libglib                   2.86.1               he69a767_2    conda-forge
libhwloc                  2.12.1          default_h48b22c3_1002    conda-forge
libiconv                  1.18                 h23cfdf5_2    conda-forge
libintl                   0.25.1               h493aca8_0    conda-forge
libintl-devel             0.25.1               h493aca8_0    conda-forge
libjpeg-turbo             3.1.2                hc919400_0    conda-forge
liblzma                   5.8.1                h39f12f2_2    conda-forge
liblzma-devel             5.8.1                h39f12f2_2    conda-forge
libogg                    1.3.5                h48c0fde_1    conda-forge
libopenvino               2025.2.0             h56e7ac4_1    conda-forge
libopenvino-arm-cpu-plugin 2025.2.0             h56e7ac4_1    conda-forge
libopenvino-auto-batch-plugin 2025.2.0             he81eb65_1    conda-forge
libopenvino-auto-plugin   2025.2.0             he81eb65_1    conda-forge
libopenvino-hetero-plugin 2025.2.0             h273c05f_1    conda-forge
libopenvino-ir-frontend   2025.2.0             h273c05f_1    conda-forge
libopenvino-onnx-frontend 2025.2.0             h6386500_1    conda-forge
libopenvino-paddle-frontend 2025.2.0             h6386500_1    conda-forge
libopenvino-pytorch-frontend 2025.2.0             hec049ff_1    conda-forge
libopenvino-tensorflow-frontend 2025.2.0             hee62d61_1    conda-forge
libopenvino-tensorflow-lite-frontend 2025.2.0             hec049ff_1    conda-forge
libopus                   1.5.2                h48c0fde_0    conda-forge
libpng                    1.6.50               h280e0eb_1    conda-forge
libprotobuf               6.31.1               h658db43_2    conda-forge
librsvg                   2.60.0               h5c55ec3_0    conda-forge
libsndfile                1.2.2                h9739721_1    conda-forge
libsqlite                 3.51.0               h8adb53f_0    conda-forge
libtiff                   4.7.1                h4030677_1    conda-forge
libusb                    1.0.29               hbc156a2_0    conda-forge
libvorbis                 1.3.7                h81086ad_2    conda-forge
libvpx                    1.14.1               h7bae524_0    conda-forge
libvulkan-loader          1.4.328.1            h49c215f_0    conda-forge
libwebp-base              1.6.0                h07db88b_0    conda-forge
libxml2                   2.15.1               h9329255_0    conda-forge
libxml2-16                2.15.1               h0ff4647_0    conda-forge
libzlib                   1.3.1                h8359307_2    conda-forge
mad                       0.15.1b              hbdafb3b_1    conda-forge
mpg123                    1.32.9               hf642e45_0    conda-forge
ncurses                   6.5                  h5e97a16_3    conda-forge
openh264                  2.6.0                hb5b2745_0    conda-forge
openssl                   3.6.0                h5503f6c_0    conda-forge
pango                     1.56.4               h875632e_0    conda-forge
pcre2                     10.46                h7125dd6_0    conda-forge
pip                       25.3               pyh8b19718_0    conda-forge
pixman                    0.46.4               h81086ad_1    conda-forge
pugixml                   1.15                 hd3d436d_0    conda-forge
python                    3.12.12         h18782d2_1_cpython    conda-forge
readline                  8.2                  h1d1bf99_2    conda-forge
sdl2                      2.32.56              h248ca61_0    conda-forge
sdl3                      3.2.26               h919df07_0    conda-forge
setuptools                80.9.0             pyhff2d567_0    conda-forge
shaderc                   2025.4               h1a5098f_0    conda-forge
snappy                    1.2.2                hada39a4_1    conda-forge
sox                       14.4.2            h267fbf7_1020    conda-forge
spirv-tools               2025.4               ha7d2532_0    conda-forge
svt-av1                   3.1.2                h12ba402_0    conda-forge
tbb                       2022.3.0             h66ce52b_1    conda-forge
tk                        8.6.13               h892fb3f_3    conda-forge
wheel                     0.45.1             pyhd8ed1ab_1    conda-forge
x264                      1!164.3095           h57fd34a_2    conda-forge
x265                      3.5                  hbc6ce65_3    conda-forge
zlib                      1.3.1                h8359307_2    conda-forge
zstd                      1.5.7                h6491c7d_2    conda-forge

*torch* modules installed using pip:
pytorch-lightning==2.5.6
torch==2.7.1
torchaudio==2.7.1
torchinfo==1.8.0
torchmetrics==1.8.2

Other modules installed using pip:
absl-py==2.3.1
aiofiles==24.1.0
aiohappyeyeballs==2.6.1
aiohttp==3.13.2
aiosignal==1.4.0
annotated-doc==0.0.4
annotated-types==0.7.0
anyio==4.11.0
anytree==2.13.0
arrow==1.4.0
attrs==25.4.0
audioread==3.1.0
black==24.10.0
brotli==1.2.0
certifi==2025.11.12
cffi==2.0.0
cfgv==3.4.0
chardet==5.2.0
charset-normalizer==3.4.4
chevron==0.14.0
click==8.1.8
clipdetect==0.1.4
coloredlogs==15.0.1
contourpy==1.3.3
coverage==7.12.0
cycler==0.12.1
decorator==5.2.1
deepdiff==8.6.1
diff-cover==9.7.2
distlib==0.4.0
dnspython==2.8.0
docopt==0.6.2
editdistance==0.8.1
einops==0.5.0
email-validator==2.3.0
et-xmlfile==2.0.0
-e file:///Users/tessierm/sandbox/TxT2Speech/EveryVoice_2025-11-18
fastapi==0.121.2
ffmpy==1.0.0
filelock==3.20.0
flake8==7.3.0
fonttools==4.60.1
frozenlist==1.8.0
fsspec==2025.10.0
g2p==2.2.2
gitlint-core==0.19.1
gradio==5.49.1
gradio-client==1.13.3
grapheme==0.6.0
greenlet==3.2.4
groovy==0.1.2
grpcio==1.76.0
h11==0.16.0
hf-xet==1.2.0
httpcore==1.0.9
httpx==0.28.1
huggingface-hub==1.1.4
humanfriendly==10.0
identify==2.6.15
idna==3.11
iniconfig==2.3.0
ipatok==0.4.2
isort==7.0.0
jinja2==3.1.6
joblib==1.5.2
jsonschema==4.25.1
jsonschema-specifications==2025.9.1
kiwisolver==1.4.9
lazy-loader==0.4
librosa==0.11.0
lightning==2.5.6
lightning-utilities==0.15.2
llvmlite==0.45.1
loguru==0.6.0
lxml==6.0.2
markdown==3.10
markdown-it-py==4.0.0
markupsafe==3.0.3
matplotlib==3.9.4
mccabe==0.7.0
mdurl==0.1.2
merge-args==0.1.5
mpmath==1.3.0
msgpack==1.1.2
multidict==6.7.0
munkres==1.1.4
mypy==1.18.2
mypy-extensions==1.1.0
networkx==3.5
nltk==3.9.1
nodeenv==1.9.1
numba==0.62.1
numpy==1.26.4
openpyxl==3.1.5
orderly-set==5.5.0
orjson==3.11.4
packaging==25.0
pandas==2.3.3
panphon==0.20.0
pathspec==0.12.1
pep440==0.1.2
pillow==11.3.0
pip @ file:///home/conda/feedstock_root/build_artifacts/pip_1762776163052/work
platformdirs==4.5.0
playwright==1.56.0
pluggy==1.6.0
pooch==1.8.2
pre-commit==4.4.0
prompt-toolkit==3.0.52
propcache==0.4.1
protobuf==4.25.8
pycodestyle==2.14.0
pycountry==22.3.5
pycparser==2.23
pydantic==2.7.4
pydantic-core==2.18.4
pydub==0.25.1
pyee==13.0.0
pyflakes==3.4.0
pygments==2.19.2
pympi-ling==1.71
pyparsing==3.2.5
pysdtw==0.0.5
pytest==7.4.4
python-dateutil==2.9.0.post0
python-multipart==0.0.20
python-slugify==5.0.0
pytz==2025.2
pyworld-prebuilt==0.3.4.4
pyyaml==6.0.3
questionary==1.10.0
readalongs==1.2.1
referencing==0.37.0
regex==2025.11.3
requests==2.32.5
rich==14.2.0
rpds-py==0.29.0
ruff==0.14.5
safehttpx==0.1.7
scikit-learn==1.7.2
scipy==1.16.3
semantic-version==2.10.0
setuptools==80.9.0
sh==2.2.2
shellingham==1.5.4
simple-term-menu==1.5.2
six==1.17.0
sniffio==1.3.1
soundfile==0.13.1
soundswallower==0.6.5
soxr==1.0.0
starlette==0.49.3
sympy==1.14.0
tabulate==0.9.0
tensorboard==2.20.0
tensorboard-data-server==0.7.2
termcolor==3.2.0
text-unidecode==1.3
threadpoolctl==3.6.0
tomlkit==0.13.3
tqdm==4.67.1
typer==0.20.0
typer-slim==0.20.0
types-pyyaml==6.0.12.20250915
types-requests==2.32.4.20250913
types-setuptools==80.9.0.20250822
types-tabulate==0.9.0.0
types-tqdm==4.67.0.20250809
typing-extensions==4.15.0
tzdata==2025.2
unicodecsv==0.14.1
urllib3==2.5.0
uvicorn==0.38.0
virtualenv==20.35.4
wcwidth==0.2.14
websockets==15.0.1
webvtt-py==0.4.2
werkzeug==3.1.3
wheel==0.45.1
yarl==1.22.0
yaspin==3.3.0
</pre>
</details>

### More info

[freeze.txt](https://github.com/user-attachments/files/23611564/freeze.txt)
